collectors/ebpf.plugin/README.md

```diff
@@ -39,18 +39,96 @@ section for details.
 
 ## Charts
 
-The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with three sub-menus: **File**, **VFS**, and
-**Process**. All the charts in this section update every second. The collector stores the actual value inside of its
-process, but charts only show the difference between the values collected in the previous and current seconds.
+The eBPF collector creates an **eBPF** menu in the Agent's dashboard along with two sub-menus: **Socket**, and
+**Process**. The plugin also creates charts on different menus, like System Overview, Memory, Disks, Filesystem, 
+Mount Points and Applications. All the charts in this section update every second.
+The collector stores the actual value inside of its process, but charts only show the difference between the values
+collected in the previous and current seconds.
 
-### File
+### System overview
 
-This group has two charts demonstrating how software interacts with the Linux kernel to open and close file descriptors.
+Not all charts within the System Overview menu are enabled by default, because they add around 100ns overhead for each 
+function call, this number is small for a human perspective, but the functions are called many times creating an impact 
+on host.
+See the [configuration](#configuration) section for details about how to enable them.
+
+#### Swap
+
+Inside the swap submenu the eBPF plugin creates the chart `swapcalls`; this chart is displaying when processes are calling
+functions `swap_readpage` and `swap_writepage`, which are functions responsible for doing IO in swap memory.
+
+### Memory
+
+In the memory submenu the eBPF plugin creates two submenus **page cache** and **synchronization** with the following
+organization:
+
+* Page Cache
+  * Page cache ratio
+  * Dirty pages
+  * Page cache hits
+  * Page cache misses
+* Synchronization
+  * File sync
+  * Memory map sync
+  * File system sync
+  * File range sync
+
+#### Page cache ratio
+
+The chart `cachestat_ratio` shows how processes are accessing page cache. In a normal scenario, we expect values around 
+ 100%, which means that the majority of the work on the machine is processed in memory. To calculate the ratio Netdata
```

Comment:
There are many places with whitespaces at the beginning of strings or at the end, like in these lines. Please remove them.

Reply:
Thanks, I removed.